rustls/src/suites.rs

```diff
@@ -214,6 +214,51 @@ pub(crate) fn compatible_sigscheme_for_suites(
         .any(|&suite| suite.usable_for_signature_algorithm(sigalg))
 }
 
+/// Secrets for transmitting/receiving data over a TLS session. These can be
```

Comment:
For docstrings, the first line should be a self-contained short description. If you have more, add an empty line after that, then describe the additional details.